tox.ini

```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = {py34, py35, py36, py37}
+envlist = {py34, py35, py36, py37}-nameko{2.6, 2.7, 2.8, 2.9, 2.10, 2.11}
 skipsdist = True
 
 [testenv]
 whitelist_externals = make
 usedevelop = true
 extras = dev
+deps =
+    nameko{2.6,2.7}: pytest<3.3.0
+    nameko{2.6,2.7}: eventlet<0.22.0
+    nameko{2.8,2.9,2.10,2.11}: pytest==4.3.0
```

Comment:
Is this line really needed?

If we do not pin it here then it'll use the `pytest` version pinned in the `dev` extra requires (the same one `4.3.0`), which is what we probably want.

If we ever upgrade that `pytest` requirement in the future and that makes any of this Nameko versions (`nameko{2.8,2.9,2.10,2.11}`) break then we can come back here and pin specific `pytest` versions to specific Nameko versions.

This way, we avoid duplicating this dependency in two places until there's a good reason for it.

What do you think?

Reply:
Oh, that's a good point!